### ISSUE TITLE
"Serialized transaction not found" error

### ISSUE THREAD

**Issue by jspence425** [MyCryptoHQ/MyCrypto#1992]
Probably needs a little investigating to figure out where this is from and why it happens.

Source: https://twitter.com/Crypt0Kin/status/1012110046011052033

Maarten confirmed it has happened to him but he was unable to reproduce.

Crypt0Kin said the following: “**Happens when I try to send a similar tx twice to the same address, I got around the normal error message by hitting the request payment tab and then hitting the send payment tab again but this error message comes up when I do it like that. It works fine if logout and back in however**. ... it is just more alarming than anything else, since the red backgrounds are associated with anti-viruses stopping people from going on malicious websites”

![screen shot 2018-06-27 at 4 15 48 pm](https://user-images.githubusercontent.com/26891718/42004761-1bae4f48-7a26-11e8-931e-f4fcf5edfce1.png)

**ADDITIONAL REQUEST:**
"No probs! Tell them make that error screen more friendly to! That thing nearly gave me a heart attack :D"

**Comment by jspence425** [comment]
User confirms he was using his keystore file when this error occurred.

**Comment by jspence425** [comment]
These are the steps that the user followed that resulted in the error occurring:

1. I sent .05 eth to an address with 1 gwei
2. I tried sending .05 again to the same address with 1 gwei but got the transaction broadcasted error.
3. To get around this I hit the "request payment" tab and then hit the "send ether" tab after
4. I tried sending again and then got the red screen. 
5. It was fixed by reloading the page and re-entering my keystore
